examples/with-nanostores/src/pages/index.astro

```diff
@@ -8,39 +8,39 @@ import AdminsPreact from '../components/AdminsPreact.jsx';
 // Full Astro Component Syntax:
 // https://github.com/snowpackjs/astro/blob/main/docs/core-concepts/astro-components.md
 ---
+
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Astro</title>
 
-    <link rel="icon" type="image/svg+xml" href="/favicon.svg">
+    <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
 
-    <link rel="stylesheet" href="/style/global.css">
-    <link rel="stylesheet" href="/style/home.css">
+    <link rel="stylesheet" href="/style/global.css" />
+    <link rel="stylesheet" href="/style/home.css" />
```

Comment:
Did you do this or is this done by a format tool? I think we prefer omitting the slash here since it's not required by HTML and we are trying to be closer to HTML than JSX.

Reply:
Yeah that must have been one of my code formatter plugins, I'll revert that